default/python/AI/freeorion_tools/_freeorion_tools.py

```diff
@@ -428,25 +428,26 @@ def get_named_real(name: str) -> float:
     return 1.0
 
 
-def get_game_rule_int(name: str) -> int:
+def get_game_rule_int(name: str, default: int = 0) -> int:
```

Comment:
Should not we preserve the old default value?
```suggestion
def get_game_rule_int(name: str, default: int = 1) -> int:
```